components/eventing-controller/pkg/backend/jetstreamv2/mocks/ConnectionInterface.go

```diff
@@ -0,0 +1,81 @@
+// Code generated by mockery v2.14.0. DO NOT EDIT.
```

Comment:
how does one generate this file?
Please add some `go:generate` comment on the source